src/Toml/Bi/Combinators.hs

```diff
@@ -143,6 +146,11 @@ word :: Key -> TomlCodec Word
 word = match _Word
 {-# INLINE word #-}
 
+-- | Codec for word8 values.
+word8 :: Key -> TomlCodec Word8
+word8 = match _Word8
+{-# INLINE word8 #-}
```

Comment:
I guess this should be reexported too :slightly_smiling_face:

Reply:
Right, missed that one